packages/augur-ui/src/modules/create-market/get-template.ts

```diff
@@ -123,87 +186,172 @@ export const getTemplateReadableDescription = (template: Template) => {
-              {
-                id: 3,
-                type: TemplateInputType.ADDED_OUTCOME,
-                placeholder: `No Winner`,
-              },
-              {
-                id: 4,
-                type: TemplateInputType.SUBSTITUTE_USER_OUTCOME,
-                placeholder: `Over ({[2] + .5})`,
-              },
-              {
-                id: 5,
-                type: TemplateInputType.SUBSTITUTE_USER_OUTCOME,
-                placeholder: `Under ({[2] - .5})`,
-              },
-            ],
+            inputs: [],
+            inputsType: TemplateInputTypeNames.OVER_UNDER,
             resolutionRules: [
               `If the game is not played or is NOT completed for any reason, the market should resolve as "No Winner".`,
             ],
           },
         ],
       },
+      [AMERICAN_FOOTBALL]: {
+        templates: [
+          {
+            templateId: `fb-teamVsteam`,
+            marketType: YES_NO,
+            question: `Will the [0] win vs the [1], Estimated schedule start time: [2]?`,
+            example: `Will the NY Giants win vs. the New England Patriots, Estimated schedule start time: Sept 19, 2019 1:00 pm EST`,
+            inputs: [],
+            inputsType: TemplateInputTypeNames.TEAM_VS_TEAM_BIN,
+            resolutionRules: [
+              `Include Regulation and Overtime`,
+              `If the game ends in a tie, the market should resolve as "NO' as Team A did NOT win vs team B`,
+            ],
+          },
+          {
+            templateId: `fb-teamVsteam-point`,
+            marketType: YES_NO,
+            question: `Will the [0] win vs the [1] by [2] or more points, Estimated schedule start time: [3]`,
+            example: `Will the NY Giants win vs. the New England Patriots by 3 or more points, Estimated schedule start time: Sept 19, 2019 1:00 pm EST`,
+            inputs: [],
+            inputsType: TemplateInputTypeNames.TEAM_VS_TEAM_POINTS_BIN,
+            resolutionRules: [
+              `Include Regulation and Overtime`,
+              `If the game ends in a tie, the market should resolve as "NO' as Team A did NOT win vs team B`,
+            ],
+          },
+          {
+            templateId: `fb-teamVsteam-point-comb`,
+            marketType: YES_NO,
+            question: `Will the [0] & [1] score [2] or more combined points, Estimated schedule start time: [3]`,
+            example: `Will the NY Giants & the New England Patriots score 44 or more combined points, Estimated schedule start time: Sept 19, 2019 1:00 pm EST`,
+            inputs: [],
+            inputsType: TemplateInputTypeNames.TEAM_VS_TEAM_POINTS_BIN,
+            resolutionRules: [
+              `Include Regulation and Overtime`,
+              `If the game ends in a tie, the market should resolve as "NO' as Team A did NOT win vs team B`,
+            ],
+          },
+        ],
+      },
     },
   },
 };
+
+const inputes = {
+  [TemplateInputTypeNames.TEAM_VS_TEAM_POINTS_BIN]: [
```

Comment:
why are these types if they are only used once

Reply:
I suspect they will be used multiple times.